apps/parse/lib/parse/calendar_attributes.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Parse.CalendarAttributes do
+  @moduledoc """
+  Parser for GTFS calendar_attributes.txt
+  """
+  @behaviour Parse
+  defstruct [
```

Comment:
Structs should be defined in the `models` app.

Reply:
The `calendar_attributes` is only used in one place: it gets passed to `state/service.ex`, which uses it to construct the `service` that it stores, but it does not directly store the `calendar_attributes` in the state. Including the struct here was following the pattern of `calendar_dates`. Should there still be a `model/calendar_attributes`?

Reply:
Fair enough.